x-pack/plugins/security_solution/cypress/urls/navigation.ts

```diff
@@ -69,3 +71,6 @@ export const hostDetailsUrl = (hostName: string) =>
 export const userDetailsUrl = (userName: string) => `/app/security/users/${userName}/allUsers`;
 
 export const exceptionsListDetailsUrl = (listId: string) => `${EXCEPTIONS_LIST_URL}/${listId}`;
+
+export const internalAlertingSnoozeRule = (ruleId: string) =>
```

Comment:
Since this file refers to frontend navigation routes, I would place this method somewhere else.

I realize there's no central file within the Cypress folder that hold all API routes, so maybe we can create one?

A new `x-pack/plugins/security_solution/cypress/tasks/api_calls/routes.ts` file can be an option.

Reply:
Right, I moved it to a separate file.